packages/daft/daft.0.0.3/opam

```diff
@@ -0,0 +1,55 @@
+opam-version: "2.0"
+name: "daft"
+maintainer: "unixjunkie@sdf.org"
+authors: "Francois Berenger"
+homepage: "https://github.com/UnixJunkie/daft"
+bug-reports: "https://github.com/UnixJunkie/daft/issues"
+dev-repo: "git+https://github.com/UnixJunkie/daft.git"
+license: "CeCILL-C"
+build: [
+  [make]
+]
+install: [
+  ["cp" "daft" "daft_ds" "daft_mds" bin]
+]
+remove: [
+  ["rm" "-f" "%{bin}%/daft" "%{bin}%/daft_ds" "%{bin}%/daft_mds"]
+]
+flags: light-uninstall
+depends: [
+  "ocaml"
+  "obuild" {build}
+  "dolog" {>= "4.0.0"}
+  "batteries"
+  "fileutils"
+  "zmq"
```

Comment:
```suggestion
  "zmq" {>= "5.0.0"}
```
Does it requires zmq 5.0.0 or something like that? It fails on OCaml 4.02 and 4.03 with the following error message:
```
#=== ERROR while compiling daft.0.0.3 =========================================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.02.3 | pinned(https://github.com/UnixJunkie/daft/archive/v0.0.3.tar.gz)
# path                 ~/.opam/4.02/.opam-switch/build/daft.0.0.3
# command              /usr/bin/make
# exit-code            2
# env-file             ~/.opam/log/daft-22-84e04e.env
# output-file          ~/.opam/log/daft-22-84e04e.out
### output ###
# obuild configure
# Configuring DAFT-0.1...
# error: missing dependency 'zmq'
# make: *** [Makefile:12: config] Error 9
```